aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -100,7 +100,15 @@ Tensor& mul_(Tensor& self, const Tensor& other) {
   return native::mul_out(self, self, other);
 }
 
+// Basic checking for all sub functions.
+static inline void sub_check(const Tensor& self, const Tensor& other) {
+  TORCH_CHECK(self.scalar_type() != kBool && other.scalar_type() != kBool,
```

Comment:
this doesn't look correct.  `~` is not the correct suggestion for `boolTensor - boolTensor` (there are two boolean operands!), but it is temporarily for "1 - boolTensor", which is intended to invert the values.

Reply:
How about this:
```
if both operands are bool:
   your error message, but with `^` instead as the suggestion.
if one operand is bool:
   "Subtraction, the `-` operator, with a bool tensor is not supported.  If you are trying to invert a mask, use the `~` or `bitwise_not()` operator instead."
```